integration-tests/chopsticks/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "e2e",
+  "version": "0.0.1",
+  "description": "chopsticks integration tests",
+  "main": "src/index.ts",
+  "type": "module",
+  "repository": "git@github.com:KILTprotocol/kilt-node.git",
+  "author": "[\"KILT <info@kilt.io>\"]",
+  "license": "MIT",
+  "dependencies": {
+    "@acala-network/chopsticks": "0.9.12",
+    "@acala-network/chopsticks-testing": "0.9.12",
+    "@polkadot/api": "^10.11.2",
+    "@types/node": "^20.11.30",
+    "bn.js": "^5.2.1",
+    "typescript": "*"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.13.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^8.0.1",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-typescript": "^18.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-standard-with-typescript": "^43.0.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-promise": "^6.0.0",
+    "prettier": "^3.2.5",
+    "ts-node": "^10.9.2",
+    "tsx": "^4.7.1",
+    "vitest": "^1.4.0"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "clean": "rm -rf ./db && cargo build",
```

Comment:
Why do we `cargo build`? If this is necessary, I think it's enough if we build the required runtime, with `cargo build -p spiritnet-runtime`.

Reply:
Good point: [48626b5](https://github.com/KILTprotocol/kilt-node/pull/614/commits/48626b53ebcdb7e1a8c66170da8c678e479e55b3)